app/components/elements/Pagination.js

```diff
@@ -94,13 +97,25 @@ Pagination.propTypes = {
   id: PropTypes.string.isRequired,
   ...PaginationProps,
   ...BoxProps,
+  controlLabels: PropTypes.shape({
+    first: PropTypes.string.isRequired,
+    last: PropTypes.string.isRequired,
+    previous: PropTypes.string.isRequired,
+    next: PropTypes.string.isRequired,
+  }),
   variant: PropTypes.oneOf(['default', 'condensed']),
   buttonVariant: PropTypes.oneOf(['pagination', 'paginationLight']),
 };
 
 Pagination.defaultProps = {
   variant: 'default',
   buttonVariant: 'pagination',
+  controlLabels: {
+    first: t('First'),
+    last: t('Last'),
+    previous: t('Previous'),
+    next: t('Next'),
+  }
```

Comment:
Missing trailing comma here: this is causing the unit tests to fail.
```suggestion
  },
```